.mailmap

```diff
@@ -1,3 +1,4 @@
 # This file sets the canonical name for contributors to the repository.
 # Documentation: https://git-scm.com/docs/gitmailmap
 Amethyst Reese <amethyst@n7.gg> <john@noswap.com>
+Willow Chargin <wchargin@gmail.com>
```

Comment:
Would it be possible to put your entry before Amethyst Reese please? we tried to alphabetically sort ACKS so let's do it for `.mailmap` as well. TiA.

Reply:
Ah, you want it by _last_ name. Done.

Reply:
Thanks!

> you want it by last name

Yes (well.. it was a bit hard to know that it was by last name considering there was only a single entry....). I find it a bit easier to have the same ordering in those files (it doesn't matter which, it's just easier if similar files follow the same ordering logic).